packages/components/src/presentation-components/tree/controlled/TreeReloader.ts

```diff
@@ -56,10 +55,7 @@ export function reloadTree(
 ): Observable<TreeModelSource> {
   const modelSource = new TreeModelSource();
   const nodeLoader = new TreeReloader(dataProvider, modelSource, pageSize, treeModel, itemsRange);
-  return nodeLoader.reloadTree().pipe(
-    endWith(modelSource),
-    finalize(() => isIDisposable(nodeLoader) && /* c8 ignore next */ nodeLoader.dispose()),
-  );
+  return nodeLoader.reloadTree().pipe(endWith(modelSource));
```

Comment:
@saskliutas, I removed the `finalize` here, since `TreeReloader` seems to never be disposable (c8 ignore kind of confirms that). Am I missing something or is it okay?